src/app/components/menu/MenuSlide.tsx

```diff
@@ -0,0 +1,110 @@
+'use client';
+
+import { AvatarFallback } from '@radix-ui/react-avatar';
+import * as VisuallyHidden from '@radix-ui/react-visually-hidden';
+import { LogOut, UserCircle2 } from 'lucide-react';
+import { User } from 'next-auth';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState, type FC } from 'react';
+
+import { Avatar, AvatarImage } from '@/components/ui/avatar';
+import { Sheet, SheetContent, SheetHeader, SheetTitle, SheetTrigger } from '@/components/ui/sheet';
+
+import { deleteAccount, signOutAuth } from '@/app/components/menu/actions';
+import { NavMenu } from '@/app/components/nav/NavMenu';
+import { SYSTEM_VERSION } from '@/app/utils/siteSettings';
+
+interface Props {
+  currentUser:
+    | (User & {
+        isLicense: boolean;
+      })
+    | undefined;
+}
```

Comment:
これでSession.userの型が取れるっぽいのでいかがでしょうか！
Sessionは'next-auth'からimportしています！
```suggestion
interface Props {
  currentUser: Session["user"]
}
```